ooasp/run.py

```diff
@@ -60,49 +72,92 @@ def _get_cautious(ctl, project=False):
     ctl.assign_external(Function("check_potential_cv"), False)
     ctl.configuration.solve.models = "0"
     ctl.configuration.solve.enum_mode = "cautious"
-    ctl.configuration.solve.opt_mode = "optN"
-    if project:
-        ctl.configuration.solve.project = "project"
+    # if project:
```

Comment:
Please remove commented-out code entirely, or add a comment explaining why it is only temporarily commented-out. Please do this for all commented-out lines of code.